docs/webanalytics/overview.md

```diff
@@ -21,12 +21,19 @@ Follow these simple steps to get started with Web Analytics.
 ## Install Statsig script on your website
 
 ```html
-<script
-  src="https://cdn.jsdelivr.net/npm/statsig-web-analytics/dist/index.min.js?apikey=[YOUR-API-KEY]">
+<script src="https://cdn.jsdelivr.net/npm/@statsig/js-client@latest/build/statsig-js-client+session-replay+web-analytics.min.js"></script>
+<script>
+  const { StatsigClient, runStatsigAutoCapture, runStatsigSessionReplay } = window.Statsig;
+  
+  // put your client sdk key here - "client-XXXX"
+  const client = new StatsigClient("[YOUR-API-KEY]", {});
+  client.initializeAsync();
```

Comment:
Have you tested what happens if you initializeAsync before calling runStatsigFoo?